### DISCUSSION TITLE
How to install on AWS Ubuntu VM?

### DISCUSSION THREAD

**Discussion by grafpaper1987** [wg-easy/wg-easy#645] [General]
Hi there. As the title states, I'm struggling to get these installed on an AWS Ubuntu instance. I follow the instructionss provided, including defining the WG_HOST and PASSWORD variables, but I keep receiving the following error when I go to run the container:

"Error: WG_HOST Environment Variable Not Set!
    at /app/lib/WireGuard.js:34:17
    at processTicksAndRejections (internal/process/task_queues.js:95:5)"

Any ideas?

**Comment by eandersons** [comment]
Full `docker run` command or content of Docker Compose file that you used would increase the chances to help you find the issue.
